js/main.js

```diff
@@ -210,7 +213,7 @@ var attentionScum = function(vocabulary) {
 var buildQuote = function(vocabulary) {
     var quote = '';
 
-    if (_.random(9) < 8) { // 80% chance
+    if (_.random(1, 100) < 85) {
```

Comment:
I think this should be `<=`, but I could use the extra pair of eyes. I think this makes it 1..84, but that's only 84%. If it were `<=`, 86..100 would be 15 numbers. I literally had to count this out on my toes...